### ISSUE TITLE
.field-label size modifiers are not working

### ISSUE THREAD

**Issue by guillecro** [jgthms/bulma#580]
### Overview of the problem

This is about the Bulma **CSS framework** 
 I'm using Bulma **version** [0.4.0]

### Description

The size modifiers for `.field-label` are not working on my screen..

```
<div class="field is-horizontal">
    <div class="field-label is-large">
        <label class="label">Nombre de la Sala</label>
    </div>
    <div class="field-body">
        <div class="field">
            ( . . . )
        </div>
    </div>
</div>
```
Hede is a gif!
![Gif](https://zippy.gfycat.com/FearfulMediocreAndeancat.gif)

In HD: [gfycat.com/FearfulMediocreAndeancat](gfycat.com/FearfulMediocreAndeancat)

I love Bulma <3

**Comment by Technologeek** [comment]
How are you using bulma ? From external css files or a CDN ? If you're using a cdn like this one https://cdnjs.cloudflare.com/ajax/libs/bulma/0.4.0/css/bulma.css chances are,they have not been committed with the latest additions.
For example,
Check the code for css in the CDN file and compare with the code for 0.4 in the repo's css file.You will notice that field-label's other properties are not reflected on the cdn.

_CSS file from the cdn_ 
![cdn1](https://cloud.githubusercontent.com/assets/3764685/24099807/31f6d5e2-0d97-11e7-819f-5d7590950e7d.PNG)

_CSS file from the repo_
![cdn22](https://cloud.githubusercontent.com/assets/3764685/24099811/35400a84-0d97-11e7-89ab-e251a888d5f8.PNG)

Try using the modifiers by running the css offline and see if that works.I could be wrong though.

**Comment by guillecro** [comment]
Thanks @Technologeek for your reply, i use npm to update bulma's module.
So basically the bulma module gets updated to 0.4.0 but the css rules for field.label size modifiers are missing. Maybe it was a lil mistake, who knows.. I will wait until its added to the official repo!
Thanks again~

**Comment by jgthms** [comment]
The latest version is always on `master` in this repository.
Npm is for releases.
The fix will probably be in `0.4.1`.
